python/ray/experimental/client/common.py

```diff
@@ -27,16 +27,17 @@ def __call__(self, *args, **kwargs):
                         "Use {self._name}.remote method instead")
 
     def remote(self, *args, **kwargs):
-        if self._raylet_remote_func is not None:
-            return self._raylet_remote_func.remote(*args, **kwargs)
-        return call_remote(self, *args, **kwargs)
+        return ray.call_remote(self, *args, **kwargs)
```

Comment:
Hmm why not make this a member of this class instead of `ray`?

Reply:
Because this class can't contain the API (it needs the ray trampoline depending on which end is calling this method)